src/containers/signup-confirmation/signup-confirmation-pf.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```

Comment:
The name of .tsx files should be in TitleCase.
so SignupConfirmationPF, instead of signup-confirmation-pf